tools/packaging/packaging.psm1

```diff
@@ -1710,14 +1713,321 @@ function CreateNugetPlatformFolder
+            $deps.Add([tuple]::Create([tuple]::Create('id', 'Microsoft.PowerShell.ConsoleHost'), [tuple]::Create('version', $PackageVersion))) > $null
+            $deps.Add([tuple]::Create([tuple]::Create('id', 'Microsoft.PowerShell.Security'), [tuple]::Create('version', $PackageVersion))) > $null
+            $deps.Add([tuple]::Create([tuple]::Create('id', 'System.Management.Automation'), [tuple]::Create('version', $PackageVersion))) > $null
+            foreach($packageInfo in (Get-ProjectPackageInformation -ProjectName $fileBaseName))
+            {
+                $deps.Add([tuple]::Create([tuple]::Create('id', $packageInfo.Name), [tuple]::Create('version', $packageInfo.Version))) > $null
+            }
+            $deps.Add([tuple]::Create([tuple]::Create('id', 'Microsoft.WSMan.Management'), [tuple]::Create('version', $PackageVersion))) > $null
+            $deps.Add([tuple]::Create([tuple]::Create('id', 'Microsoft.PowerShell.Commands.Diagnostics'), [tuple]::Create('version', $PackageVersion))) > $null
+            $deps.Add([tuple]::Create([tuple]::Create('id', 'Microsoft.Management.Infrastructure.CimCmdlets'), [tuple]::Create('version', $PackageVersion))) > $null
+        }
+
+        'Microsoft.PowerShell.Security' {
+            $deps.Add([tuple]::Create([tuple]::Create('id', 'System.Management.Automation'), [tuple]::Create('version', $PackageVersion))) > $null
+        }
+
+        'Microsoft.WSMan.Management' {
+            $deps.Add([tuple]::Create([tuple]::Create('id', 'System.Management.Automation'), [tuple]::Create('version', $PackageVersion))) > $null
+            $deps.Add([tuple]::Create([tuple]::Create('id', 'Microsoft.WSMan.Runtime'), [tuple]::Create('version', $PackageVersion))) > $null
+            foreach($packageInfo in (Get-ProjectPackageInformation -ProjectName $fileBaseName))
+            {
+                $deps.Add([tuple]::Create([tuple]::Create('id', $packageInfo.Name), [tuple]::Create('version', $packageInfo.Version))) > $null
+            }
+        }
+
+        'Microsoft.WSMan.Runtime' {
+            ## No dependencies
+        }
+
+        'System.Management.Automation' {
+            $deps.Add([tuple]::Create([tuple]::Create('id', 'Microsoft.PowerShell.CoreCLR.Eventing'), [tuple]::Create('version', $PackageVersion))) > $null
+            foreach($packageInfo in (Get-ProjectPackageInformation -ProjectName $fileBaseName))
+            {
+                $deps.Add([tuple]::Create([tuple]::Create('id', $packageInfo.Name), [tuple]::Create('version', $packageInfo.Version))) > $null
+            }
+        }
+    }
+
+    $srcFilePackagePath = Join-Path $PackagePath $fileBaseName
+
+    Write-Verbose -Verbose "New-ILNugetPackageFromSource: Creating nuget package for file: $FileName from source path: $srcFilePackagePath"
+    Write-Verbose -Verbose -Message "$(Get-ChildItem -Path $srcFilePackagePath)"
```

Comment:
Are usually prefer something more like this:
```powershell
PS> (ls).foreach({write-verbose -verbose -message $_})
VERBOSE: /Users/james/src/github/forks/jameswtruher/UnixCompleters/out
VERBOSE: /Users/james/src/github/forks/jameswtruher/UnixCompleters/PSUnixUtilCompleters
VERBOSE: /Users/james/src/github/forks/jameswtruher/UnixCompleters/tests
VERBOSE: /Users/james/src/github/forks/jameswtruher/UnixCompleters/build.ps1
VERBOSE: /Users/james/src/github/forks/jameswtruher/UnixCompleters/completions.gif
VERBOSE: /Users/james/src/github/forks/jameswtruher/UnixCompleters/LICENSE
VERBOSE: /Users/james/src/github/forks/jameswtruher/UnixCompleters/OnStart.ps1
VERBOSE: /Users/james/src/github/forks/jameswtruher/UnixCompleters/PSUnixUtilCompleters.psd1
VERBOSE: /Users/james/src/github/forks/jameswtruher/UnixCompleters/README.md
```
because this is a little messy
```powershell
PS> write-verbose -verbose "$(ls)"                          
VERBOSE: /Users/james/src/github/forks/jameswtruher/UnixCompleters/out /Users/james/src/github/forks/jameswtruher/UnixCompleters/PSUnixUtilCompleters /Users/james/src/github/forks/jameswtruher/UnixCompleters/tests /Users/james/src/github/forks/jameswtruher/UnixCompleters/build.ps1 /Users/james/src/github/forks/jameswtruher/UnixCompleters/completions.gif /Users/james/src/github/forks/jameswtruher/UnixCompleters/LICENSE /Users/james/src/github/forks/jameswtruher/UnixCompleters/OnStart.ps1 /Users/james/src/github/forks/jameswtruher/UnixCompleters/PSUnixUtilCompleters.psd1 /Users/james/src/github/forks/jameswtruher/UnixCompleters/README.md
```

Reply:
This is actually not needed any more and I'll remove it.